dashboards/kubernetes/node.json

```diff
@@ -0,0 +1,300 @@
+        "metrics": [
+          {
+            "name": "node_fs_available_bytes",
+            "fields": [
+              {
+                "field": "GAUGE"
+              }
+            ],
+            "tags": [
+              {
+                "key": "node",
+                "value": "*"
+              }
+            ]
+          },
+          {
+            "name": "node_fs_used_bytes",
+            "fields": [
+              {
+                "field": "GAUGE"
+              }
+            ],
+            "tags": [
+              {
+                "key": "node",
+                "value": "*"
+              }
+            ]
+          }
+        ],
+        "type": "timeseries"
+      },
+      {
+        "title": "Node Network Traffic Received",
+        "description": "node_network_rx_bytes",
+        "line_label": "%name% %node%",
+        "display": "LINE",
+        "format": "size",
+        "format_input": "byte",
+        "draw_null_as_zero": true,
+        "metrics": [
+          {
+            "name": "node_network_rx_bytes",
+            "fields": [
+              {
+                "field": "GAUGE"
+              }
+            ],
+            "tags": [
+              {
+                "key": "node",
+                "value": "*"
+              }
+            ]
+          }
+        ],
+        "type": "timeseries"
+      },
+      {
+        "title": "Node Network Traffic Transmitted",
+        "description": "node_network_tx_bytes",
+        "line_label": "%name% %node%",
+        "display": "LINE",
+        "format": "size",
+        "format_input": "byte",
+        "draw_null_as_zero": true,
+        "metrics": [
+          {
+            "name": "node_network_tx_bytes",
+            "fields": [
+              {
+                "field": "GAUGE"
+              }
+            ],
+            "tags": [
+              {
+                "key": "node",
+                "value": "*"
+              }
+            ]
+          }
+        ],
+        "type": "timeseries"
+      },
+      {
+        "title": "Node Inodes Usage",
+        "description": "node_fs_inodes_free & node_fs_inodes_used vs node_fs_inodes",
+        "line_label": "%name% %node%",
+        "display": "LINE",
+        "format": "number",
+        "draw_null_as_zero": true,
+        "metrics": [
+          {
+            "name": "node_fs_inodes",
+            "fields": [
+              {
+                "field": "GAUGE"
+              }
+            ],
+            "tags": [
+              {
+                "key": "node",
+                "value": "*"
+              }
+            ]
+          },
+          {
+            "name": "node_fs_inodes_free",
+            "fields": [
+              {
+                "field": "GAUGE"
+              }
+            ],
+            "tags": [
+              {
+                "key": "node",
+                "value": "*"
+              }
+            ]
+          },
+          {
+            "name": "node_fs_inodes_used",
```

Comment:
Let's use tags for different states like free and used instead of reporting them as different metrics. We do this for other (host) metrics too. It would help that we don't have to show the full metric name then for every line in the graph, freeing up valuable space in the hover box.

For example:

- Metric name: `node_fs_inodes`
- Tags:
  - `state`
    - Values:
      - `free`
      - `used`

I also see this in some other graphs in this dashboard. We should update those as well.